site/create.md

```diff
@@ -19,6 +19,9 @@ The Create Domain Tool understands three domain types: `WLS`, `RestrictedJRF`, a
 
 To have the Create Domain Tool run RCU, simply add the `-run_rcu` argument to the previous command line and the RCU schemas will be automatically created.  Be aware that when the tool runs RCU, it will automatically drop any conflicting schemas that already exist with the same RCU prefix prior to creating the new schemas!
 
+It is also possible to specify the connection information in the model instead of 
+using the command line.  This is especially easier for complicated jdbc connection string such as RAC or Oracle Autonomous Transaction Processing Cloud Service databases.  For information of how to use it, refer to [Specifying RCU connection information in the model](rcuinfo.md)
```

Comment:
complicated jdbc connection string such as -> complicated JDBC connection strings, such as

Reply:
For information of how to use it -> For information on how to use it